source/cli.js

```diff
@@ -79,6 +79,7 @@ process.on('SIGINT', () => {
 		{
 			...cli.flags,
 			confirm: true,
+			exists: !isAvailable,
```

Comment:
I need this, and it is returned from the `ui()` step, so makes sense to add it here as well.